### PR TITLE
patch(CLI) descriptive function names in CLI

### PR DESCRIPTION
**Pull request by islean**
## Description

Some functions in the CLI were non-descriptive. Commands were left as is, while function names are renamed to be more understandable.

### Changed

- Renamed CLI to have clearer names.

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b 1991-descriptive-cli-functions -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/cli/delete/cases.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from cg.store import Store
 from cg.store.models import Family, Sample
 
-from .case import case
+from .case import delete_case
```
Comment (henrikstranneheim) [inline cg/cli/delete/cases.py:7]: Use absolute import

**Review by henrikstranneheim — APPROVED** [review]
Fantastic 👍!

**Review by seallard — APPROVED** [review]
Nice 👍

**Comment by Vince-janv** [comment]
```
θ67° [vincent.janvid@hasta:~] [S_base] 3m42s $ cg get sample ACC11950A9
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Demux.samples to refer to viewonly relationship Sample.demuxes should include sync_backref=False set on the Demux.samples relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Sample.demuxes to refer to viewonly relationship Demux.samples should include sync_backref=False set on the Sample.demuxes relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
+------------+---------------+------------+---------------+--------------------+------------+-------------+
| Sample     | Name          | Customer   | Application   | State              | Priority   | External?   |
|------------+---------------+------------+---------------+--------------------+------------+-------------|
| ACC11950A9 | 2023-15756-03 | cust002    | WGSPCFC030    | Ordered 2023-05-04 | standard   | No          |
+------------+---------------+------------+---------------+--------------------+------------+-------------+
+--------------+----------+------------+------------+-----------+----------+
| Family       | Name     | Customer   | Priority   | Panels    | Action   |
|--------------+----------+------------+------------+-----------+----------|
| casualimpala | F0053540 | cust002    | standard   | OMIM-AUTO | NA       |
+--------------+----------+------------+------------+-----------+----------+
+------------+----------+----------+
| Sample     | Mother   | Father   |
|------------+----------+----------|
| ACC11950A9 |          |          |
+------------+----------+----------+
```

```
⌂22.04 θ71° [vincent.janvid@hasta:~] [S_base] 130 $ cg add user -c cust000 --admin test@scilifelab.se "test user"
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Demux.samples to refer to viewonly relationship Sample.demuxes should include sync_backref=False set on the Demux.samples relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Sample.demuxes to refer to viewonly relationship Demux.samples should include sync_backref=False set on the Sample.demuxes relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
User added: test@scilifelab.se (354)
```

```
θ73° [vincent.janvid@hasta:~] [S_base] 22s $ cg upload scout justhusky -r
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
----------------- UPLOAD -----------------
----------------- SCOUT -----------------------
Fetching family object
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Demux.samples to refer to viewonly relationship Sample.demuxes should include sync_backref=False set on the Demux.samples relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Sample.demuxes to refer to viewonly relationship Demux.samples should include sync_backref=False set on the Sample.demuxes relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
Called undefined __fields__ on HousekeeperAPI, please wrap
----------------- CREATE CONFIG -----------------------
Create load config
Generate scout load config
Fetch latest version from bundle justhusky
Found pipeline mip-dna
Generate load config for mip case
Including cohorts to scout load config
Adding phenotype groups to scout load config
Adding phenotype terms to scout load config
Fetch latest version from bundle justhusky
Fetching files from bundle justhusky
Fetching files with tags in [mip-config]
Fetching files from version 140077
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/query.py:2577: SAWarning: Pathed join target File.version has already been joined to; skipping
  "been joined to; skipping" % prop
Fetch latest version from bundle justhusky
Fetching files from bundle justhusky
Fetching files with tags in [qc-metrics,deliverable]
Fetching files from version 140077
Fetch latest version from bundle justhusky
Fetching files from bundle justhusky
Fetching files with tags in [sample-info]
Fetching files from version 140077
Including MIP specific case level files
Get file with tags {'vcf-snv-clinical'}
Get file with tags {'vcf-sv-clinical'}
Get file with tags {'vcf-snv-research'}
Get file with tags {'vcf-sv-research'}
Get file with tags {'vcf-str'}
Get file with tags {'ped', 'peddy'}
Get file with tags {'sex-check', 'peddy'}
Get file with tags {'peddy', 'ped-check'}
Get file with tags {'smn-calling'}
Include MultiQC report to case
Get file with tags {'multiqc-html'}
Include delivery report to case
Get file with tags {'delivery-report'}
Could not find any files matching the tags {'delivery-report'}
Building samples
Building sample earlycasualcaiman
Could not fetch sample earlycasualcaiman from LIMS: 404: Sample not found: earlycasualcaiman
Adding common files for sample earlycasualcaiman
Get file with tags {'bam', 'earlycasualcaiman'}
Get file with tags {'cram', 'earlycasualcaiman'}
Including MIP specific sample level files
Get file with tags {'earlycasualcaiman', 'vcf2cytosure'}
Get file with tags {'bam-mt', 'earlycasualcaiman'}
Get file with tags {'chromograph', 'autozyg', 'earlycasualcaiman'}
Get file with tags {'chromograph', 'tcov', 'earlycasualcaiman'}
Get file with tags {'chromograph', 'regions', 'earlycasualcaiman'}
Could not find any files matching the tags {'chromograph', 'regions', 'earlycasualcaiman'}
Get file with tags {'chromograph', 'sites', 'earlycasualcaiman'}
Could not find any files matching the tags {'chromograph', 'sites', 'earlycasualcaiman'}
Get file with tags {'expansionhunter', 'bam', 'earlycasualcaiman'}
Get file with tags {'expansionhunter', 'bam-index', 'earlycasualcaiman'}
Could not find any files matching the tags {'expansionhunter', 'bam-index', 'earlycasualcaiman'}
Get file with tags {'expansionhunter', 'vcf-str', 'earlycasualcaiman'}
Get file with tags {'expansionhunter', 'variant-catalog'}
Get file with tags {'mitodel', 'earlycasualcaiman'}
Building sample slowlycivilbuck
Could not fetch sample slowlycivilbuck from LIMS: 404: Sample not found: slowlycivilbuck
Adding common files for sample slowlycivilbuck
Get file with tags {'slowlycivilbuck', 'bam'}
Get file with tags {'slowlycivilbuck', 'cram'}
Including MIP specific sample level files
Get file with tags {'slowlycivilbuck', 'vcf2cytosure'}
Get file with tags {'slowlycivilbuck', 'bam-mt'}
Get file with tags {'chromograph', 'autozyg', 'slowlycivilbuck'}
Get file with tags {'chromograph', 'tcov', 'slowlycivilbuck'}
Get file with tags {'chromograph', 'regions', 'slowlycivilbuck'}
Could not find any files matching the tags {'chromograph', 'regions', 'slowlycivilbuck'}
Get file with tags {'chromograph', 'sites', 'slowlycivilbuck'}
Could not find any files matching the tags {'chromograph', 'sites', 'slowlycivilbuck'}
Get file with tags {'expansionhunter', 'bam', 'slowlycivilbuck'}
Get file with tags {'expansionhunter', 'bam-index', 'slowlycivilbuck'}
Could not find any files matching the tags {'expansionhunter', 'bam-index', 'slowlycivilbuck'}
Get file with tags {'expansionhunter', 'vcf-str', 'slowlycivilbuck'}
Get file with tags {'expansionhunter', 'variant-catalog'}
Get file with tags {'mitodel', 'slowlycivilbuck'}
Building sample hugelymodelbat
Could not fetch sample hugelymodelbat from LIMS: 404: Sample not found: hugelymodelbat
Adding common files for sample hugelymodelbat
Get file with tags {'bam', 'hugelymodelbat'}
Get file with tags {'cram', 'hugelymodelbat'}
Including MIP specific sample level files
Get file with tags {'hugelymodelbat', 'vcf2cytosure'}
Get file with tags {'hugelymodelbat', 'bam-mt'}
Get file with tags {'chromograph', 'autozyg', 'hugelymodelbat'}
Get file with tags {'chromograph', 'hugelymodelbat', 'tcov'}
Get file with tags {'chromograph', 'regions', 'hugelymodelbat'}
Get file with tags {'chromograph', 'hugelymodelbat', 'sites'}
Get file with tags {'expansionhunter', 'bam', 'hugelymodelbat'}
Get file with tags {'expansionhunter', 'bam-index', 'hugelymodelbat'}
Could not find any files matching the tags {'expansionhunter', 'bam-index', 'hugelymodelbat'}
Get file with tags {'expansionhunter', 'hugelymodelbat', 'vcf-str'}
Get file with tags {'expansionhunter', 'variant-catalog'}
Get file with tags {'mitodel', 'hugelymodelbat'}
Generate madeline output to /home/vincent.janvid/tmp/tmp7lquw6oa/madeline.xml
Generating madeline input lines
Running command /home/proj/stage/servers/resources/hasta.scilifelab.se/madeline2 --color --nolabeltruncation --outputprefix /home/vincent.janvid/tmp/tmp7lquw6oa/madeline /home/vincent.janvid/tmp/tmp76xwqojo
Found load config owner='cust000' family='justhusky' family_name='SWEDAC-2019-HG-05-Trio-NovaSeq' synopsis=None phenotype_terms=None phenotype_groups=None gene_panels=['BRAIN', 'Cardiology', 'CILM', 'CH', 'CTD', 'DIAB', 'ENDO', 'EP', 'HEARING', 'HYDRO', 'IBMFS', 'IEM', 'IF', 'mcarta', 'MHT', 'MIT', 'MOVE', 'mtDNA', 'NBS-M', 'NEURODEG', 'NMD', 'PEDHEP', 'PID', 'PIDCAD', 'RETINA', 'OMIM-AUTO', 'PANELAPP-GREEN', 'SKD', 'SOVM'] default_gene_panels=['OMIM-AUTO'] cohorts=None human_genome_build='37' rank_model_version='1.34' rank_score_threshold=5 sv_rank_model_version='1.9' analysis_date=datetime.datetime(2023, 5, 4, 18, 18, 43) samples=[ScoutMipIndividual(alignment_path='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/earlycasualcaiman_lanes_1_sorted_md.cram', analysis_type='wgs', capture_kit=None, confirmed_parent=None, confirmed_sex=None, father='0', mother='0', phenotype='unaffected', sample_id='earlycasualcaiman', sample_name='ACC5339A2--300M', sex='male', subject_id=None, tissue_type='unknown', mt_bam='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/earlycasualcaiman_lanes_1_sorted_md_subsample_MT.bam', chromograph_images=ChromographImages(autozygous='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb_rhocall_norm_af_rhoviz.earlycasualcaiman.autozyg_', coverage='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/earlycasualcaiman_lanes_1_sorted_md_tcov_', upd_regions=None, upd_sites=None), reviewer=Reviewer(alignment='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_exphun_earlycasualcaiman.bam', alignment_index=None, vcf='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_exphun_earlycasualcaiman.vcf', catalog='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/grch37_expansionhunter_variant_catalog_-5.0.0-.json'), rhocall_bed=None, rhocall_wig=None, tiddit_coverage_wig=None, upd_regions_bed=None, upd_sites_bed=None, vcf2cytosure='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_cyto.earlycasualcaiman.cgh', mitodel_file='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/earlycasualcaiman_lanes_1_sorted_md_mitodel.txt'), ScoutMipIndividual(alignment_path='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/slowlycivilbuck_lanes_1_sorted_md.cram', analysis_type='wgs', capture_kit=None, confirmed_parent=None, confirmed_sex=None, father='0', mother='0', phenotype='unaffected', sample_id='slowlycivilbuck', sample_name='ACC5339A3--300M', sex='female', subject_id=None, tissue_type='unknown', mt_bam='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/slowlycivilbuck_lanes_1_sorted_md_subsample_MT.bam', chromograph_images=ChromographImages(autozygous='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb_rhocall_norm_af_rhoviz.slowlycivilbuck.autozyg_', coverage='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/slowlycivilbuck_lanes_1_sorted_md_tcov_', upd_regions=None, upd_sites=None), reviewer=Reviewer(alignment='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_exphun_slowlycivilbuck.bam', alignment_index=None, vcf='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_exphun_slowlycivilbuck.vcf', catalog='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/grch37_expansionhunter_variant_catalog_-5.0.0-.json'), rhocall_bed=None, rhocall_wig=None, tiddit_coverage_wig=None, upd_regions_bed=None, upd_sites_bed=None, vcf2cytosure='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_cyto.slowlycivilbuck.cgh', mitodel_file='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/slowlycivilbuck_lanes_1_sorted_md_mitodel.txt'), ScoutMipIndividual(alignment_path='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/hugelymodelbat_lanes_1_sorted_md.cram', analysis_type='wgs', capture_kit=None, confirmed_parent=None, confirmed_sex=None, father='earlycasualcaiman', mother='slowlycivilbuck', phenotype='affected', sample_id='hugelymodelbat', sample_name='ACC5339A4--300M', sex='male', subject_id=None, tissue_type='unknown', mt_bam='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/hugelymodelbat_lanes_1_sorted_md_subsample_MT.bam', chromograph_images=ChromographImages(autozygous='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb_rhocall_norm_af_rhoviz.hugelymodelbat.autozyg_', coverage='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/hugelymodelbat_lanes_1_sorted_md_tcov_', upd_regions='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/hugelymodelbat_gatkcomb_rhocall_norm_af_upd.regions_', upd_sites='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/hugelymodelbat_gatkcomb_rhocall_norm_af_upd.sites_'), reviewer=Reviewer(alignment='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_exphun_hugelymodelbat.bam', alignment_index=None, vcf='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_exphun_hugelymodelbat.vcf', catalog='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/grch37_expansionhunter_variant_catalog_-5.0.0-.json'), rhocall_bed=None, rhocall_wig=None, tiddit_coverage_wig=None, upd_regions_bed=None, upd_sites_bed=None, vcf2cytosure='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_cyto.hugelymodelbat.cgh', mitodel_file='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/hugelymodelbat_lanes_1_sorted_md_mitodel.txt')] delivery_report=None coverage_qc_report=None cnv_report=None multiqc='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/multiqc_report.html' track='rare' smn_tsv='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_smn.tsv' chromograph_image_files=None chromograph_prefixes=None vcf_snv='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb_rhocall_norm_af_mt_frqf_cadd_vep_parsed.selected.vcf.gz' vcf_sv='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_comb_ann_vep_parsed_ranked.selected.vcf.gz' vcf_str='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_exphun.vcf.gz' vcf_snv_research='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb_rhocall_norm_af_mt_frqf_cadd_vep_parsed.vcf.gz' vcf_sv_research='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_comb_ann_vep_parsed_ranked.vcf.gz' peddy_ped='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb.peddy.ped' peddy_sex='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb.sex_check.csv' peddy_check='/home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/justhusky_gatkcomb.ped_check.csv' madeline='/home/vincent.janvid/tmp/tmp7lquw6oa/madeline.xml' variant_catalog=None
Set root dir to /home/proj/stage/rare-disease/cases
Saving config file to disc
Save Scout load config to /home/proj/stage/rare-disease/cases/justhusky/scout_load.yaml
Add config file to housekeeper
Adding load config /home/proj/stage/rare-disease/cases/justhusky/scout_load.yaml to Housekeeper
Fetch latest version from bundle justhusky
Could not find any files matching the tags {'scout-load-config'}
Fetching tag with name: scout-load-config
Fetching tag with name: scout-load-config
Created new bundle version dir: /home/proj/stage/housekeeper-bundles/justhusky/2023-05-04
Linked file: /home/proj/stage/rare-disease/cases/justhusky/scout_load.yaml -> /home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/scout_load.yaml
Added Scout load config to Housekeeper: /home/proj/stage/rare-disease/cases/justhusky/scout_load.yaml
----------------- UPLOAD -----------------------
Fetch latest version from bundle justhusky
Uploading case justhusky to scout
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml export cases --json --case-id justhusky
Validating case justhusky
update existing Scout case
Running command /home/proj/stage/bin/miniconda3/envs/S_scout/bin/scout --config /home/proj/stage/servers/config/hasta.scilifelab.se/scout-stage.yaml load case /home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/scout_load.yaml --update
Uploaded to scout using load config /home/proj/stage/housekeeper-bundles/justhusky/2023-05-04/scout_load.yaml
Case loaded successfully to Scout
```

```
[vincent.janvid@hasta:~] [S_base] 24s $ cg set sample -kv original_ticket 123456 ACC11952A23
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Demux.samples to refer to viewonly relationship Sample.demuxes should include sync_backref=False set on the Demux.samples relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Sample.demuxes to refer to viewonly relationship Demux.samples should include sync_backref=False set on the Sample.demuxes relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
Would change from original_ticket=845669 to original_ticket=123456 on ACC11952A23 (PB-P-00469636-CFDNA-04155154-KH20230502-C420230503)
Continue? [y/N]: y
Would set original_ticket to 123456 for ACC11952A23 in LIMS
Continue? [y/N]: n
```

```
[vincent.janvid@hasta:~] [S_base] 22s $ cg delete case wellseal
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:2231: SAWarning: Cascade settings "delete, merge, save-update" should not be combined with a viewonly=True relationship.   This configuration will raise an error in version 1.4.  Note that in versions prior to 1.4, these cascade settings may still produce a mutating effect even though this relationship is marked as viewonly=True.
  "viewonly=True." % (", ".join(sorted(non_viewonly)))
Running CG delete
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Demux.samples to refer to viewonly relationship Sample.demuxes should include sync_backref=False set on the Demux.samples relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
/home/proj/stage/bin/miniconda3/envs/S_main/lib/python3.7/site-packages/sqlalchemy/orm/relationships.py:1998: SAWarning: Setting backref / back_populates on relationship Sample.demuxes to refer to viewonly relationship Demux.samples should include sync_backref=False set on the Sample.demuxes relationship.  (this warning may be suppressed after 10 occurrences)
  (rel_b, rel_a, rel_b),
+----------+--------+------------+------------+----------+----------+
| Family   |   Name | Customer   | Priority   | Panels   | Action   |
|----------+--------+------------+------------+----------+----------|
| wellseal | 337334 | cust000    | research   | IEM      | hold     |
+----------+--------+------------+------------+----------+----------+
+-----------+----------+----------+
| Sample    | Mother   | Father   |
|-----------+----------+----------|
| ADM1136A1 |          |          |
+-----------+----------+----------+
+-----------+----------+----------+
| Sample    | Mother   | Father   |
|-----------+----------+----------|
| ADM1136A2 |          |          |
+-----------+----------+----------+
+-----------+----------+----------+
| Sample    | Mother   | Father   |
|-----------+----------+----------|
| ADM1136A3 |          |          |
+-----------+----------+----------+
+-----------+---------+------------+---------------+----------------------+------------+-------------+
| Sample    | Name    | Customer   | Application   | State                | Priority   | External?   |
|-----------+---------+------------+---------------+----------------------+------------+-------------|
| ADM1136A1 | NA12877 | cust000    | EXOSXTR100    | Sequenced 2016-03-11 | research   | No          |
+-----------+---------+------------+---------------+----------------------+------------+-------------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| H3KJ5BCXX  | hiseqga | D00410      | 2015-09-17 | No          | removed  |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| C7GA6ANXX  | hiseqga | D00450      | 2015-09-17 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| H7MFLBCXX  | hiseqga | D00410      | 2015-11-21 | No          | removed  |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| HJKMYBCXX  | hiseqga | D00410      | 2016-02-20 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| HJJYNBCXX  | hiseqga | D00450      | 2016-03-11 | No          | removed  |
+------------+---------+-------------+------------+-------------+----------+
+-----------+---------+------------+---------------+----------------------+------------+-------------+
| Sample    | Name    | Customer   | Application   | State                | Priority   | External?   |
|-----------+---------+------------+---------------+----------------------+------------+-------------|
| ADM1136A2 | NA12878 | cust000    | EXOSXTR100    | Sequenced 2016-02-20 | research   | No          |
+-----------+---------+------------+---------------+----------------------+------------+-------------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| C7GA6ANXX  | hiseqga | D00450      | 2015-09-17 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| HG7YCBCXX  | hiseqga | D00134      | 2015-12-11 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| HJKMYBCXX  | hiseqga | D00410      | 2016-02-20 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
+-----------+---------+------------+---------------+----------------------+------------+-------------+
| Sample    | Name    | Customer   | Application   | State                | Priority   | External?   |
|-----------+---------+------------+---------------+----------------------+------------+-------------|
| ADM1136A3 | NA12882 | cust000    | EXOSXTR100    | Sequenced 2016-02-20 | research   | No          |
+-----------+---------+------------+---------------+----------------------+------------+-------------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| C7GA6ANXX  | hiseqga | D00450      | 2015-09-17 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| HG7YCBCXX  | hiseqga | D00134      | 2015-12-11 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
+------------+---------+-------------+------------+-------------+----------+
| Flowcell   | Type    | Sequencer   | Date       | Archived?   | Status   |
|------------+---------+-------------+------------+-------------+----------|
| HJKMYBCXX  | hiseqga | D00410      | 2016-02-20 | No          | ondisk   |
+------------+---------+-------------+------------+-------------+----------+
Case wellseal has links: [<cg.store.models.FamilySample object at 0x7ff7f7b7acc0>, <cg.store.models.FamilySample object at 0x7ff7f7b7ad30>, <cg.store.models.FamilySample object at 0x7ff7f7b7ae10>], continue? [y/N]: y
Do you want to DELETE link: wellseal | ADM1136A1? [y/N]: y
Deleting link: wellseal | ADM1136A1
Do you want to DELETE link: wellseal | ADM1136A2? [y/N]: y
Deleting link: wellseal | ADM1136A2
Do you want to DELETE link: wellseal | ADM1136A3? [y/N]: y
Deleting link: wellseal | ADM1136A3
Can NOT delete processed sample: ADM1136A1
Sample was received: None
Sample was prepared: None
Sample was sequenced: 2016-03-11 10:39:03
Sample was delivered: None
Sample has invoice: None
Can NOT delete processed sample: ADM1136A2
Sample was received: None
Sample was prepared: None
Sample was sequenced: 2016-02-20 19:14:39
Sample was delivered: None
Sample has invoice: None
Can NOT delete processed sample: ADM1136A3
Sample was received: None
Sample was prepared: None
Sample was sequenced: 2016-02-20 19:14:39
Sample was delivered: None
Sample has invoice: None
Do you want to DELETE case: wellseal (337334)? [y/N]: y
Deleting case: wellseal
```

**Pull request merged**